install-deps.sh

```diff
@@ -395,7 +395,16 @@ else
     debian|ubuntu|devuan|elementary|softiron)
         echo "Using apt-get to install dependencies"
         if [ "$INSTALL_EXTRA_PACKAGES" ]; then
-            $SUDO apt-get install -y $INSTALL_EXTRA_PACKAGES
+            if ! $SUDO apt-get install -y $INSTALL_EXTRA_PACKAGES ; then
```

Comment:
i think the commit message does not hold anymore. as i tried to explain it in https://github.com/ceph/ceph/pull/48697#discussion_r1034672915 and the link referenced in that comment. if you are still not sure, in which part you are not convinced?

Reply:
Oh, I did update the comment but not the commit message. Let me do that now. I wish github made it easier to comment on the commit messages.